deephaven_ipywidgets/deephaven.py

```diff
@@ -20,6 +20,10 @@
 import atexit
 
 
+TABLE_TYPES = ("deephaven.table.Table", "pandas.core.frame.DataFrame", "pydeephaven.table.Table")
+FIGURE_TYPES = ("deephaven.plot.figure.Figure")
```

Comment:
Set instead of tuple

```suggestion
TABLE_TYPES = {"deephaven.table.Table", "pandas.core.frame.DataFrame", "pydeephaven.table.Table"}
FIGURE_TYPES = {"deephaven.plot.figure.Figure"}
```